bin/dataduct

```diff
@@ -13,37 +13,25 @@ from dataduct.config import logger_configuration
 import logging
 logger = logging.getLogger(__name__)
 
-CREATE_STR = 'create'
-VALIDATE_STR = 'validate'
-ACTIVATE_STR = 'activate'
-VISUALIZE_STR = 'visualize'
-DROP_STR = 'drop'
-GRANT_STR = 'grant'
-RECREATE_STR = 'recreate'
+PIPELINE = 'pipeline'
+CREATE = 'create'
+VALIDATE = 'validate'
+ACTIVATE = 'activate'
+VISUALIZE = 'visualize'
 
+CONFIG = 'config'
 CONFIG_TO_S3 = 'sync_to_s3'
 CONFIG_FROM_S3 = 'sync_from_s3'
 
-CONFIG_COMMAND = 'config'
-DATABASE_COMMAND = 'database'
-PIPELINE_COMMAND = 'pipeline'
-DEV = 'dev'
-
-formatter_class = lambda prog: RawTextHelpFormatter(prog, max_help_position=50)
-
-
-def config_actions(action, filename):
-    """Config related actions are executed in this block
-    """
-    from dataduct.config.config_actions import sync_to_s3
-    from dataduct.config.config_actions import sync_from_s3
+DATABASE = 'database'
+DROP = 'drop'
+GRANT = 'grant'
+RECREATE = 'recreate'
 
-    if action == CONFIG_TO_S3:
-        return sync_to_s3()
-    return sync_from_s3(filename)
+DEV = 'dev'
```

Comment:
This should be moved into the config file as a `default_mode` option, since not all users may have 'dev' as their default mode.

Doesn't have to be done now!